README.md

```diff
@@ -103,9 +97,8 @@ List current environment version for each microservice hosted in Kubernetes. Fur
 - `-n | --namespace` - Kubernetes namespace
 
 #### Resources
-`./resources/cosmos-upload`  
-
-Upload multiple items to a local emulator instance of Cosmos. Insert information as prompted. Currently the items uploaded match only the messages and queries schemas.
```

Comment:
Have removed the reference to Cosmos resources from the readme, but have also removed these from the resources folder since we're not currently using Cosmos. These are still on the [ffc-sfd-core](https://github.com/defra/ffc-sfd-core) repo so can always copy them back in from there.